pkg/quota/v1/install/patch_ignoredresources.go

```diff
@@ -0,0 +1,34 @@
+package install
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+func init() {
+	newIgnoredResources := map[schema.GroupResource]struct{}{
+		{Group: "extensions", Resource: "replicationcontrollers"}:                     {},
+		{Group: "extensions", Resource: "networkpolicies"}:                            {},
```

Comment:
can we remove the extensions groups here? cc @soltysh

Reply:
How would I answer this question? There isn't any detail in the commit message.

Reply:
Drop replicationcontrollers since that's not needed, networkpolicies although deprecated are still served from extensions/v1beta1 so we should keep it there for now.

Reply:
Done